### ISSUE TITLE
Disable/Delete Microsoft.PythonTools.Analyzer.exe

### ISSUE THREAD

**Issue by manstie** [microsoft/PTVS#6181]
**Describe the feature you are requesting**
Allow me to delete Microsoft.PythonTools.Analyzer.exe without an error appearing or allow me to disable the feature.

**Reasons for requesting this feature**
Whenever I have my django server running this file eventually takes up all my ram (11gb from this one .exe out of 16gb).

**Describe alternatives you've considered**
I deleted the file so it doesn't run when I run my server for testing but I get an error, and there is no way to disable it.

**Additional context**
[Window Title]
devenv.exe

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
Build: 16.6.20100.1

System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.StartSubprocessConnection(String comment, AnalysisProcessInfo& proc)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.<InitializeAsync>d__42.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.<CreateForProjectAsync>d__33.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Project.PythonProjectNode.<CreateAnalyzerAsync>d__142.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.PythonTools.Project.PythonProjectNode.<ReanalyzeProjectHelper>d__155.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Project.PythonProjectNode.<ReanalyzeProject>d__154.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Infrastructure.VSTaskExtensions.<HandleAllExceptions>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

**Comment by huguesv** [comment]
I've just merged this in, it will be in the upcoming 16.8 Preview 2.

It adds a checkbox in tools/options/python/language server to "Disable IntelliSense"

Just reload the project after checking that option, and you should no longer see the Microsoft.PythonTools.Analyzer.exe process running. The trade off is that you won't have any IntelliSense, refactoring, find references, go to definition, semantic colorization, etc.

**Comment by manstie** [comment]
cheers
